github/issues.go

```diff
@@ -55,6 +55,11 @@ func (i Issue) String() string {
 	return Stringify(i)
 }
 
+// IsPullRequest returns true if the issue is also a pull request.
```

Comment:
In fact, we might want to document what `IsPullRequest` does, so that people don't get stuck in a situation where they're not sure which of the 2 approaches they should use to check if an issue is a PR (to check `PullRequestLinks` as GitHub docs tell them to, or to use `IsPullRequest` method which they don't know what it does).

By documenting how it operates (i.e., it also uses `PullRequestLinks` value), people can then feel more confident using it because they know it does the same thing.